src/api/app/views/webui/user/_edit_dialog.html.haml

```diff
@@ -12,5 +12,10 @@
           .form-group
             = f.label(:email, 'Email:')
             = f.text_field(:email, required: true, email: true, class: 'form-control')
+          .form-group
+            = f.label(:biography) do
+              Biography:
+              %small.form-text The biography can not contain more than 250 characters.
```

Comment:
```suggestion
              %small.form-text.text-muted The biography can not contain more than 250 characters.
```

Reply:
It would make it a little less dominant

Reply:
the description is a bit overkill in my opinion, its already clear that this is related to the biography, so maybe its enough to write something like "max. 250 characters" or "Must be 1-250 characters long"

Reply:
I like to see it `text-muted`. However, if remember correctly, in the past we had the same discussion when we added this kind of small texts in other forms. At that time we decided not to make it grey, so we don't have too many font sizes/colors/weights... 

The configuration page is a good example:

![Screenshot_2020-10-19 Open Build Service](https://user-images.githubusercontent.com/2581944/96429218-fd114700-1200-11eb-8ecb-13fe497f4329.png)

Reply:
Then the result could be like:

![Screenshot_2020-10-19 Profile of scp](https://user-images.githubusercontent.com/2581944/96429895-c556cf00-1201-11eb-9fdb-3b7064c7047f.png)

Reply:
@saraycp Ah okay, I forgot this, fine for me then! :+1: